src/dash-table/components/Table/props.ts

```diff
@@ -25,6 +25,23 @@ export interface IDerivedData {
     indices: Indices;
 }
 
+export interface IViewportOffset {
+    rows: number;
+    columns: number;
```

Comment:
column offset is always 0 for now but we could virtualize horizontally eventually if needed